xla/service/cpu/tests/onednn_matmul_test.cc

```diff
@@ -1647,6 +1647,32 @@ TEST_F(MatmulTest, WeightsPrepackAndScratch) {
   )");
 }
 
+TEST_F(MatmulTest, PrepackLarge2DWeights) {
```

Comment:
Is this test actually passing in OSS? We haven't set the flag to turn the pass on, so it shouldn't pass yet right? Can we expedite the PR to fix the test and get it in first so we can actually test this and other future PRs?

Reply:
@penpornk, thanks for the feedback! I just want to make sure I understand your request correctly.

As of now, oneDNN tests (including the `PrepackLarge2DWeights` test) won't pass in OSS unless XLA is built with the `--define=build_with_onednn_async=true` flag. If that's set and the unit test is run with `XLA_FLAGS="--xla_cpu_experimental_onednn_custom_call=true"`, the test will fail on the current `main`. However, with the changes in this PR and those flags, the test should pass successfully.

Please let me know if this clarifies things or if there's another specific PR you're referring to that needs to be expedited.

Reply:
Sure. I added the flag setting (https://github.com/openxla/xla/pull/31747/commits/46bc2e644eb57fa7c482d058c76449c3de03d187) for oneDNN MatMul and Conv tests (as Conv is merged already). I will add it for Layernorm and Softmax in the Softmax PR (https://github.com/openxla/xla/pull/31745).